pkg/virt-controller/watch/vmi.go

```diff
@@ -778,6 +791,29 @@ func (c *VMIController) updateStatus(vmi *virtv1.VirtualMachineInstance, pod *k8
 	return nil
 }
 
+func isVirtHandlerAlive(clientset kubecli.KubevirtClient, nodeName string) (bool, error) {
```

Comment:
I'd change this function so it returns `err != nil` only in cases where an actual error happened. Coming to the conclusion that no virt handler is active is not an error?

Reply:
Done.